payload-schemas/schemas/marketplace_purchase/pending_change_cancelled.schema.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "marketplace_purchase$pending_change_cancelled",
+  "type": "object",
+  "required": ["action", "effective_date", "sender", "marketplace_purchase"],
+  "properties": {
+    "action": { "type": "string", "enum": ["pending_change_cancelled"] },
+    "effective_date": { "type": "string" },
+    "sender": {
```

Comment:
I think this should be a reference to the `user` common schema, as they only diff by `type`. See #308